Dockerfile

```diff
@@ -1,6 +1,13 @@
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
 WORKDIR /go/src/github.com/openshift/crd-schema-gen
 COPY . .
+ENV GO_PACKAGE github.com/openshift/crd-schema-gen
 RUN go build -o crd-schema-gen cmd/crd-schema-gen/main.go
+
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+WORKDIR /go/src/github.com/openshift/crd-schema-gen
+COPY --from=builder /go/src /go/src
+COPY --from=builder /usr/local/go /usr/local/go
```

Comment:
I'm not sure if this is the right fix, but it does fix that `Unsupported kind` error I was getting before with the second stage

@tnozicka @sttts could you review and then we can move along with https://github.com/openshift/release/pull/3428?